docs/sdk/api/sdkMiddlewareHttp.md

```diff
@@ -67,6 +68,7 @@ const client = createClient({
       includeOriginalRequest: true,
       maskSensitiveHeaderData: true,
       enableRetry: true,
+      retryCodes: [504, 'ETIMEDOUT', 'ECONNREFUSED', 503],
```

Comment:
I would have made the retryCodes a property of the retryConfig. And use a default value array with the 503 status code like:

```
retryConfig: {
   retryCodes: [503],
  ...
}
```